pdr_backend/analytics/predictoor_dashboard/dash_components/util.py

```diff
@@ -54,6 +54,23 @@ def get_predictoors_data_from_db(lake_dir: str):
 
 
 @enforce_types
+def get_user_payouts_stats_from_db(lake_dir: str):
+    return _query_db(
+        lake_dir,
+        f"""
+            SELECT 
+                "user",
+                SUM(payout - stake) AS total_profit,
+                SUM(CASE WHEN payout > 0 THEN 1 ELSE 0 END) * 100.0 / COUNT(*) AS avg_accuracy,
+                AVG(stake) AS avg_stake
+            FROM 
+                {Payout.get_lake_table_name()}
+            GROUP BY 
+                "user" 
+        """,
+    )
```

Comment:
good job with the query, the data comes in very fast!
I'm wondering if we can unite this query with the get predictoors one and have a get predictoors data query containing all the data we need, should be straight forward.